### ISSUE TITLE
Missing newline at the end of error returns

### ISSUE THREAD

**Issue by sandordargo** [tipi-build/cli#27]
I'm not completely sure if it's only with erroneous returns, but usually if a command fails, there is no newline at the end of the error message.

Example:
```
std::exception::what: Impossible to localize the working directory /home/dev/mosze-projects on the remote build
● [err] Something went wrong, for help on your issue, we would enjoy getting your report at https://tipi.build/#contactroot@fbd8a81d9ea4:/home/dev/mosze-projects# 
```

Not a big deal, but not nice.

**Comment by daminetreg** [comment]
Yes there is actually a worse bug behind this. Without the newline means we didn't call all cleanup operation that should have. Definitely a good catch. 👍

**Comment by pysco68** [comment]
Hi @sandordargo we finally managed to resolve this one. I hope you'll find v0.0.32 nicer in terms of UI consistency - we made quite a push on that end 😃 

Resolved in v0.0.32 / https://github.com/tipi-build/cli/releases/tag/v0.0.32

**Comment by sandordargo** [comment]
Definitely, it's a great improvement! Thanks, guys!
